crypto/src/signature.rs

```diff
@@ -179,6 +177,12 @@ impl<T> Ord for SignatureOf<T> {
     }
 }
 
+impl<T> core::hash::Hash for SignatureOf<T> {
+    fn hash<H: core::hash::Hasher>(&self, state: &mut H) {
+        self.0.hash(state);
+    }
+}
```

Comment:
Did `PhantomData<T>` affect hash somehow?

Reply:
no, it was to remove the lint `derive_hash_xor_eq`